### PR TITLE
Krux02 devel

### PR DESCRIPTION
**Pull request by krux02**
this is just for me

### CODE REVIEW

**Comment by krux02** [comment]
hmm, this is supposed to fail. I guess a test case needs to be addad for try.

**Comment by dom96** [comment]
Just for you? Please have some sympathy for the 282 people who are watching this repo.

**Comment by timotheecour** [comment]
I'm concerned whether pushes to this branch will trigger CI (which has limited bandwidth); I've enabled CI on my own fork to test changes without affecting nim repo

**Comment by krux02** [comment]
@timotheecour maybe I should do that, too. But isn't it the exact same CI servers that run the tests on your repo?

**Comment by genotrance** [comment]
Same servers but different quotas.

**Comment by timotheecour** [comment]
ya thats what i was gonna reply

**Comment by timotheecour** [comment]
ping @krux02 ; if that answers your question, can this be closed?

File: lib/pure/strutils.nim [hunk 1]
```diff
@@ -149,8 +149,8 @@ proc isAlphaAscii*(s: string): bool {.noSideEffect, procvar,
   ## in `s`.
   runnableExamples:
     doAssert isAlphaAscii("fooBar") == true
-    doAssert isAlphaAscii("fooBar1") == false 
-    doAssert isAlphaAscii("foo Bar") == false 
+    doAssert isAlphaAscii("fooBar1") == false
```
Comment (kaushalmodi) [inline lib/pure/strutils.nim]: @Araq Are these back now?

**Pull request merged**